tests/test_ci/TestControllers.py

```diff
@@ -459,7 +459,7 @@ def test_remove_blocked_users(self, mock_request):
             g.db.add(blocked_user)
             g.db.commit()
             self.assertNotEqual(BlockedUsers.query.filter(BlockedUsers.comment == "Bad user").first(), None)
-            c.post("/blocked_users", data=dict(user_id=1, remove=True))
+            c.post("/blocked_users/1", data=dict(blocked_user_id=1, remove=True))
```

Comment:
The blocked user id in the post data seems no longer necessary? It's not present in the form, so why would we want to test with the id in the post data? Same for the change below.